meetings/minutes/2022-02-07-TSC-Minutes.md

```diff
@@ -1,11 +1,11 @@
-# Minutes from the 2022-01-10 Laminas Technical Steering Committee Meeting
+# Minutes from the 2022-02-07 Laminas Technical Steering Committee Meeting
 
 - When: 2022-02-07 at 19:00 UTC, until 19:57 UTC
 - Where: Laminas Slack #tsc-meeting channel
 - Attending:
   - Abdul Malik Ikhsan
   - Aleksei Khudiakov
-  - Frank Bruchner
+  - Frank Brückner
```

Comment:
I don't know why I have the wrong spelling in my head, but I consistently do this! Sorry, @froschdesign !